lib/commons/aria/ariadescribedby-text.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Get the accessible name based on aria-describedby
+ *
+ * @deprecated Do not use Element directly. Pass VirtualNode instead
+ * @param {VirtualNode|Element} element
+ * @param {Object} context
+ * @property {Bool} inLabelledByContext Whether or not the lookup is part of aria-describedby reference
+ * @property {Bool} inControlContext Whether or not the lookup is part of a native label reference
+ * @property {Element} startNode First node in accessible name computation
+ * @property {Bool} debug Enable logging for formControlValue
+ * @return {string} Concatenated text value for referenced elements
+ */
+function ariadescribedbyText(element, context = {}) {
+  const { vNode } = axe.utils.nodeLookup(element);
```

Comment:
here in the param we are passing virtualNode itself no ? 
Do we need to extract vNode again after passing it to nodeLookup ?

Reply:
This logic was implemented by axe only in the aria labelled text function and similary in many other functions like isFocusable and all. So kept it.